+siibra/+core/Parcellation.m

```diff
@@ -35,53 +35,45 @@
             regions = webread(parcellation_json.links.regions.href);
             
             % store graph
-            parcellation.Graph = Parcellation.createParcellationTree(parcellation, regions);
-            
+            parcellation.regionTree = siibra.core.Parcellation.createParcellationTree(parcellation, regions);
             
             %parcellation.Spaces = table(string({spaces_subset.Name}).', spaces_subset.', 'VariableNames', {'Name', 'Space'});
         end
-        %getters
-        function space_table = spaceTable(obj)  
-            space_table = table(string({obj.Spaces.Name}).', obj.Spaces.', 'VariableNames', {'Name', 'Space'});
-        end
-        function region_table = findRegion(obj, region_name_query)
-            region_table = obj.Graph.Nodes(contains(obj.Graph.Nodes.Name, region_name_query), :);
+       
+        function region_names = findRegion(obj, region_name_query)
+            region_names = obj.regionTree.Nodes(contains(obj.regionTree.Nodes.Name, region_name_query), 1);
         end
         function region = decodeRegion(obj, region_name_query)
             region_table = obj.findRegion(region_name_query);
             assert(height(region_table) == 1, "query was not unambiguous!")
             region = region_table.Region(1);
         end
         function region = getRegion(obj, region_name_query)
-            nodeId = obj.Graph.findnode(region_name_query);
-            region = obj.Graph.Nodes.Region(nodeId);
+            nodeId = obj.regionTree.findnode(region_name_query);
+            region = obj.regionTree.Nodes.Region(nodeId);
         end
-        function children = getChildrenNames(obj, region_name)
-            nodeId = obj.Graph.findnode(region_name);
-            childrenIds = obj.Graph.successors(nodeId);
-            children = obj.Graph.Nodes.Name(childrenIds);
+        function children = getChildrenRegions(obj, region_name)
```

Comment:
I suggest to name this "getChildRegions", the double plural sounds strange to me